bin/dsh

```diff
@@ -1468,6 +1524,10 @@ case $1 in
 	vmstat)
 		vmstat
 		;;
+	ssh-add)
```

Comment:
This command is for people, who use multiple ssh keys.

`dsh ssh-add -l` - list ssh identities available in ssh-agent
`dsh ssh-add -D` - remove all
`dsh ssh-add` - add the default identity (`id_rsa`)